proto/buf/alpha/breaking/v1/config.proto

```diff
@@ -0,0 +1,51 @@
+// Copyright 2020-2021 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package buf.alpha.breaking.v1;
+
+// Config represents the breaking change configuration for a module. The rule IDs and categories are defined
+// by the rules_version and apply across the config. The rules_version is independent of the version of
+// the package. The package version refers to the config shape, the rules_version indicates which
+// rule IDs and catetgories should be used.
+//
+// The rule IDs and categories are not encoded as enums in this package because we may want to support custom rule
+// IDs and categories in the future. Callers will need to resolve the rule IDs and categories strings.
+message Config {
+  // rules_version represents the version of the breaking change rule IDs and categories that should be used with this config.
+  string rules_version = 1;
+  // use lists the rule IDs and/or categories that are included in the breaking change check.
+  repeated string use = 2;
```

Comment:
`use_ids` etc?

Reply:
Offline decision: `use_rule_ids` (and etc. for following fields)